programs/extcuraengine.hs

```diff
@@ -143,7 +143,6 @@ layers print fs = catMaybes <$> rawContours
     allIntersections :: ℝ -> [(Point2,Point2)]
     allIntersections zLayer = catMaybes $ triIntersects zLayer <$> fs
     zs = [zOf . fst <$> triPoints | triPoints <- sidesOf <$> fs ] `using` parListChunk (div (length fs) (fromFastℕ threads)) rseq
-    zmax :: ℝ
```

Comment:
Did you remove this intentionally?  I find it way more readable if all `where` blocks have empty lines between all definitions, and explicit type signatures on all names introduced.

Reply:
I'll re-add this later. noted.